spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe Api::V1::UsersController do
+  before(:each) do
+    allow(Figaro.env).to receive(:transitland_datastore_auth_token) { 'THISISANOTHERKEY' }
+    @request.env['HTTP_AUTHORIZATION'] = 'Token token=THISISANOTHERKEY'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.